### PR TITLE
Allow configure IgnoreCase in settings

### PR DESCRIPTION
**Pull request by leolplex**
Hello @StefH  @JanuszRybinski,

This is the PR related with this #692, the idea is allow not case sensitive in Headers, Query Params, Request Body and examples.

Regards.

### CODE REVIEW

**Review by StefH — CHANGES_REQUESTED** [review]
It's better to move this setting (**IgnoreCase**) to `WireMockOpenApiParserSettings.cs` to that it can be modified by user.

Default value can be set to `true`

**Comment by leolplex** [comment]
@StefH,

you're right it's better idea, done!

File: src/WireMock.Net.OpenApiParser/Settings/WireMockOpenApiParserSettings.cs [hunk 1]
```diff
@@ -36,5 +36,33 @@ public class WireMockOpenApiParserSettings
         /// Are examples generated dynamically?
         /// </summary>
         public bool DynamicExamples { get; set; } = false;
+
+        /// <summary>
+        /// Is headers case sensitive? (default is true).        
+        /// true = no case sensitive;
```
Comment (StefH) [inline src/WireMock.Net.OpenApiParser/Settings/WireMockOpenApiParserSettings.cs:42]: No need to add these two lines:
```
        /// true = no case sensitive;
        /// false = case sensitive 
```

You can remove these.

**Review by StefH — CHANGES_REQUESTED** [review]

File: src/WireMock.Net.OpenApiParser/Settings/WireMockOpenApiParserSettings.cs (hunk 1 above)
Comment (leolplex) [reply src/WireMock.Net.OpenApiParser/Settings/WireMockOpenApiParserSettings.cs]: @StefH  Done!

**Review by StefH — APPROVED** [review]

**Pull request merged**